product.json

```diff
@@ -29,9 +29,7 @@
 	"documentationUrl": "https://go.microsoft.com/fwlink/?linkid=862277",
 	"commit": "9ca6200018fc206d67a47229f991901a8a453781",
 	"date": "2017-12-15T12:00:00.000Z",
-	"extensionsGallery": {
-		"serviceUrl":"https://raw.githubusercontent.com/Microsoft/sqlopsstudio/release/extensions/extensionsGallery.json"
-	},
-	"recommendedExtensions": [
-	]
+	"recommendedExtensions": [],
+	"updateUrl": "test",
+	"quality": "insider"
```

Comment:
Did you mean to leave these here?

Reply:
I'll remove it


---
In reply to: [174277101](https://github.com/Microsoft/sqlopsstudio/pull/902#discussion_r174277101) [](ancestors = 174277101)